ResearchKit/Common/ORKScaleSlider.m

```diff
@@ -119,12 +119,19 @@ - (CGSize)intrinsicContentSize {
 - (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent *)event {
     BOOL pointInside = NO;
     if (_vertical) {
-        // In vertical mode, we need to ignore the touch area for the needed extra width
-        const CGFloat desiredSliderWidth = 44.0;
-        const CGFloat actualWidth = [self bounds].size.width;
-        const CGFloat centerX = actualWidth / 2;
-        if (fabs(point.y - centerX) < desiredSliderWidth / 2) {
-            pointInside = [super pointInside:point withEvent:event];
+        
+        if (_textChoices) {
+            if (point.y > self.bounds.size.width/2) {
```

Comment:
I'd made the comparison
`point.y > (self.bounds.size.width - DesiredSliderWidth)/2`, to avoid a bit of leeway if you are touching very near the left of the slider.

Extract constant value from below.